### DISCUSSION TITLE
Roadmap

### DISCUSSION THREAD

**Discussion by prologic** [nuxui/nuxui#10] [Ideas]
Hey it's me again 👋

Have re-built all your latest samples again on both Desktop (macOS) and Mobile (iOS):

- fomater doesn't work on iOS
- gallery crashes on iOS
- counter works great!
- hello works great!
- widgets: still some weirdness going on. I see you removed the Editor?

Antyway... Going through what you've done, I _think_ the project would really benefit from having a well-defined Roadmap.

I'm sure you have an idea in mind of where you're trying to go? 🤗

**Comment by adamkali** [comment]
Hi there! I am new to this project and just saw it on Reddit and I really like it! I really want to help this project grow. 

So I want to get a couple of questions/suggestions  down, and anyone can feel free to shoot down/ask for clarifications.

For context, I am a C#/React developer professionally and a Go/RN developer personally, but react native is there for necessity for a lack of meaningfully better options. 

So, with that being said, are some some questions/expectations/suggestions I can think of if the top of my head that I know others will ask about. And if these questions have been answered feel free to either reiterate here (so questions can be in one place) or post links. 

## Questions 
- [ ] What is the design philosophy behind Nux? (Or how should a end-developer using Nux design their projects using Nux?)

- [ ] How does Nux handle state? (Is it like React? Svelte? Flutter? Or new Nux state philosophy?

- [ ] Is or will all of css be supported? 

- [ ] Will Nux use concurrency in Go to help performance? If so how?

- [ ] Is this another Tauri/Electron? Or something different?

- [ ] Does the styling features transpile straight to CSS?

## Expectations
- [ ]  What should I expect performance wise?

- [ ] What do I get from switching from React/Flutter?

- [ ] Can I use other libraries with Nux? Think tailwind, or Scss? (This can be important!) 

- [ ] What is the memory imprint and bundle size for the Client?

## Suggestions
- [ ] Will Nux do more with animations?

- [ ] Will Nux have a [expo](https://docs.expo.dev/get-started/installation/) for devs that do not have access to a MacOS machine?

- [ ] How do I opt in or out of platforms. Is everything on by default?

I hope that some of all of these questions can be answered as I am really excited about this framework.

**Comment by fodedoumbouya** [comment]
I am new to this project and I have being playing with the code and some examples too 
I really like the idea and the project so I wanted to know if I can participate to the project by bring some help for the grow of it 
I am full stack dev mobile from Flutter/dart, Golang etc .. 
hoping this project will great thing for Go community
